### DISCUSSION TITLE
How do I use the same keyboard shorts as vs code

### DISCUSSION THREAD

**Discussion by fayazara** [zed-industries/zed#6563] [Feature Requests]
I am really enjoying Zed, but one issue is, I love my vscode shortcut bindings, is there a way to move the shortcuts and use the same in Zed?

**Comment by JosephTLyons** [comment]
Unfortunately, we don't have any mechanism to import custom key bindings from another text editor.  Our defaults mostly mirror VS Code's default bindings, but anything custom has to be manually added to the `keymap.json` file (`zed: open keymap` in the command palette).

**Reply by folaoluwafemi** [reply]
importing keyboard shortcuts would be really amazing for users (like me) who are using zed for the first time and have a tight relationship with some keybinding...

I feel like I'm in a foreign land where they speak new languages and I have to ask how things are done for every step I take.

it would atleast be nice if I could find or create key binding extensions too tho, or perhaps contribute the feature???
anyways zed is a pretty cool editor :D

**Reply by jrnkng** [reply]
Agree, shouldn't be a hard functionality to implement at all but would make a massive difference. Heavily doubting my switch now

**Reply by avDec25** [reply]
it's difficult to move to zed because of this very reason. It's a deal breaker.
At least let me have my Zed windows keymap to be imported in Zed MacOS. 
I copied default Zed-Keymap from Windows to MacOS, that also failed

**Comment by linuxiamo** [comment]
You should add by default those of Notepad++.

**Comment by avDec25** [comment]
STOP Everything you are working on Related to ZED editor, 
and add Keymap import feature from various editors, already existing on a User's system. At least add VS-CODE import.
